packages/nimble-components/src/anchor-tree-item/styles.ts

```diff
@@ -32,10 +34,15 @@ export const styles = css`
         --ni-private-tree-item-nested-width: 0;
     }
 
+    :host([disabled]) {
+        color: ${bodyDisabledFontColor};
+        cursor: default;
+    }
+
     .control {
         display: flex;
         text-decoration: none;
-        color: ${bodyFontColor};
+        color: currentcolor;
```

Comment:
Setting `color` to `currentcolor` seems like a misuse of that setting. Shouldn't it just be `inherit`?